### ISSUE TITLE
请问一下，qwen 的vllm部署和原始的部署出来的结果存在不一致，结果是经过多次反复验证的。请问有没有解决的方案

### ISSUE THREAD

**Issue by qingjiaozyn** [QwenLM/Qwen#1074]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by jklj077** [comment]
Consistency in results across various frameworks isn't guaranteed due to inherent variations in their design. Nevertheless, according to internal evaluations, when operating under identical inference conditions, the disparity between outcomes produced by `transformers` and `vllm` should not have a substantial impact on the final results.

Please note that since both these packages are continuously evolving, the validity of this statement could vary at present. Unfortunately, without specific details such as the model used, versions of the packages employed, etc., it's not feasible to delve into any potential discrepancies that might exist currently.

**Comment by WangJianQ-cmd** [comment]
我使用的原始的部署方式（需要注意vllm版本库）：
python -m fastchat.serve.controller
python -m fastchat.serve.vllm_worker --model-path Qwen/Qwen-72B-Chat --trust-remote-code --tensor-parallel-size 2 --gpu-memory-utilization 0.98 --dtype bfloat16
python -m fastchat.serve.openai_api_server --host '0.0.0.0' --port 8889

**Comment by qingjiaozyn** [comment]
@WangJianQ-cmd  vllm版本用的0.3.0版本，启动参数跟你的一样

**Comment by qingjiaozyn** [comment]
> Consistency in results across various frameworks isn't guaranteed due to inherent variations in their design. Nevertheless, according to internal evaluations, when operating under identical inference conditions, the disparity between outcomes produced by `transformers` and `vllm` should not have a substantial impact on the final results.
> 
> Please note that since both these packages are continuously evolving, the validity of this statement could vary at present. Unfortunately, without specific details such as the model used, versions of the packages employed, etc., it's not feasible to delve into any potential discrepancies that might exist currently.

Whether this difference can also be optimized. I tested it and found that the accuracy of VLLM QWEN-14B dropped from 90% to 80% compared to the original QWEN-14B. It has already made an impact, and the accuracy has dropped too much
